pkgs/development/python-modules/beanquery/default.nix

```
@@ -0,0 +1,53 @@
+{
+  lib,
+  beancount,
+  click,
+  buildPythonPackage,
+  fetchFromGitHub,
+  python-dateutil,
+  pytestCheckHook,
+  setuptools,
+  tatsu,
+}:
+buildPythonPackage rec {
+  pname = "beanquery";
+  version = "0.1.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "beancount";
+    repo = "beanquery";
+    rev = "v0.1.0";
```

```suggestion
    rev = "refs/tags/v${version}";
```
refs/tags to avoid confusion with same-name branches, ${version} so only one place needs to be updated